src/locales/en.json

```diff
@@ -105,5 +108,14 @@
     "errorStack": "Error stack trace:",
     "errorStack_loading": "Loading data. please wait...",
     "sceneContent": "Scene content:"
+  },
+  "roomDialog": {
+    "desc_intro": "You can create a session below to collaborate with anyone you want.",
+    "desc_privacy": "Don't worry, the session uses end-to-end encryption, so whatever you draw will stay private. Not even our server will be able to see what you come up with.",
+    "desc_start": "To begin, click the button below. (Will open a room in a new tab.)",
```

Comment:
Is there a reason why we're opening it in a new tab? This seems a bit silly to me.

Reply:
First, I wasn't doing it. But then realized we'd have to account for you loosing whatever scene you're currently working on. Or maybe I'm missing something?

(not sure how the i18n you linked to is related)

Reply:
I think I wasn't aware that the init doesn't do anything with the current scene. Mhm, so I guess we can use the same tab.

Reply:
If so, should I make it so that you can start/stop the session, which will set/reset the URL?

Reply:
I would expect that when you click on the live sharing, it keeps the current scene and makes it live. It is explicit on your part to share it to other people so they can join, so if you don't want the current scene, you can clear the scene.

If you stop sharing, then the current scene remains but you don't broadcast / receive updates from people.

Reply:
Yes. So will make `start session` to init session using current scene, and `stop session` will simply destroy the connection, but keep current scene state.

And if you click on `start session` again after that, it will initialize a new session, different from previous.

Agreed?

Reply:
Thinking more about this, I'm not sure that we really need an explicit Start button. We could have clicking on the live icon open the popup with the link and explanation. Then clicking on the share button again stops sharing / leave the room.

This way it's only one dialog and less steps.

Reply:
Would love thoughts on this.

Reply:
> Yes. So will make start session to init session using current scene, and stop session will simply destroy the connection, but keep current scene state.

> And if you click on start session again after that, it will initialize a new session, different from previous.

Agreed

Reply:
When the session is started, we want to update the tab's url. If we start the session automatically, and user decides she doesn't want to start the session after all, what then.

Also, overloading the RoomDialog button like you suggest doesn't feel intuitive. What if you don't want the 2nd click on it to stop sharing?

Reply:
(didn't see your 3rd comment before I posted my last)

Reply:
> user decides she doesn't want to start the session after all, what then.

They click on the icon again.

The main reason you want a confirm dialog is when the action is not reversible or has a big impact. In this case, creating a shared room does nothing until someone else joins. So if you don't want to be in a live mode anymore, just click the icon again. The stakes are very low.

Reply:
Ooh.. so I'm not sure if I was supposed to make that rewrite :).

Reply:
I still think the overhead is negligible, and more user friendly.

Reply:
Also note that users may not realize that the URL and the link they're supposed to share is identical.

Reply:
As mentioned below, I'm fine with this as is. Please get it working and ship it :)